llvm/lib/Target/RISCV/RISCVInstrInfo.h

```diff
@@ -359,5 +359,31 @@ struct PseudoInfo {
 
 } // end namespace RISCVVPseudosTable
 
+namespace RISCVVInversePseudosTable {
```

Comment:
We can move this to a common header (MCTargetDesc/RISCVMCTargetDesc.h?) to remove duplicated code in MCA.

Reply:
Done